README.md

```diff
@@ -374,6 +374,13 @@ is `False`.
 
 Escaped newlines at the end of hash comments add the following line to the comment. Apache's native config parser processes hash comments this way. Set to `False` by default.
 
+### disableselfclosingtags
```

Comment:
The other name for self-closing tags is "empty-element tag". Not sure which is better understood / more canonical.

Just noting this, I am good with any of these names.

Reply:
Not sure either! I just did some digging and did find "empty-element tag" used in the [wikipedia page for XML](https://en.wikipedia.org/wiki/XML#Key_terminology) -- so let's go ahead with that.